pytika/api.py

```diff
@@ -54,6 +54,9 @@ def get_meta(self, file: BufferedReader, key=None) -> dict:
         self.handle_errors(res)
 
         metadata = res.json()
+
+        if hasattr(file, "seek"):
```

Comment:
I like the fact that you checked for the attribute. I think you're right to assume that the user doesn't really care if the seek fails as long as the rest of the operation succeeded, and this is just a nice-to-have for them.

One additional idea you could implement is catch any potential exceptions here in case the user has some weird file type. But I don't know how I feel about this one, at this point we're protecting the user a bit too much maybe.

```python
    if hasattr(file, "seek"):
        try:
            file.seek(0)  # Reset filereader for reusability
        except Exception:
            pass  # Ignore the error if the file object does not support seeking
```